nixos/tests/all-tests.nix

```diff
@@ -96,6 +96,7 @@ in {
   blockbook-frontend = handleTest ./blockbook-frontend.nix {};
   blocky = handleTest ./blocky.nix {};
   boot = handleTestOn ["x86_64-linux" "aarch64-linux"] ./boot.nix {};
+  bootspec = handleTestOn ["x86_64-linux" "aarch64-linux"] ./bootspec.nix {};
```

Comment:
Not quire sure aarch64-linux is correct here; ofborg seems to have choked on it (maybe it can't run _all_ of the tests?)

Reply:
https://logs.nix.ci/?attempt_id=4d0b879d-f2a0-4110-987e-ff629ee45925&key=nixos%2Fnixpkgs.207157 :thinking:

Reply:
classical aarch64 shenigans, I would not be too shocked to proceed even if it fails on aarch64 for now.
On our side, SecureBoot will need a non trivial thing to happen on aarch64, see https://github.com/msekletar/sbsigntool/issues/2

Reply:
```
error: builder for '/nix/store/hnazyp7qdy2y9ybggh90cdgbssmibwwf-nixos-boot-disk.drv' failed with exit code 1;
       last 10 log lines:
       > Creating new GPT entries in memory.
       > The operation has completed successfully.
       > mkfs.fat 4.2 (2021-01-31)
       > Hidden (32) does not match sectors (63)
       > insmod: ERROR: missing filename.
       > insmod: ERROR: missing filename.
       > insmod: ERROR: missing filename.
       > updating GRUB 2 menu...
       > cannot copy /nix/store/bhwwg3fxq9h5fmvjbgrm0rdvjd64hliw-initrd-linux-5.15.83/initrd to /boot/kernels/bhwwg3fxq9h5fmvjbgrm0rdvjd64hliw-initrd-linux-5.15.83-initrd.tmp: No space left on device
       > [  866.799073] reboot: Power down
       For full logs, run 'nix log /nix/store/hnazyp7qdy2y9ybggh90cdgbssmibwwf-nixos-boot-disk.drv'.
error: 1 dependencies of derivation '/nix/store/375gl7c1ckw0g5xg4dr6975rii2gzhfm-run-nixos-vm.drv' failed to build
error: 1 dependencies of derivation '/nix/store/v8vgby2gn4sxx1lmajz60w2b1312h085-nixos-vm.drv' failed to build
error: 1 dependencies of derivation '/nix/store/9lsp0dkb1q22diq0w7scpsk85mnlic0q-nixos-test-driver-legacy-boot-with-bootspec.drv' failed to build
error: 1 dependencies of derivation '/nix/store/ha354wwziksns1xivczrynd0hy6h8yn1-vm-test-run-legacy-boot-with-bootspec.drv' failed to build
error: build of '/nix/store/0q1wlr77xxsrbprz6s4m59zcpyl0lp8d-vm-test-run-systemd-boot-with-bootspec.drv', '/nix/store/20w40lqwkqhaa627f9iq506gdxl70jkb-vm-test-run-bootspec-with-specialisation.drv', '/nix/store/ha354wwziksns1xivczrynd0hy6h8yn1-vm-test-run-legacy-boot-with-bootspec.drv', '/nix/store/if6dzhhaaa5n1d6qpmv4v5a849s88fab-vm-test-run-bootspec-with-extensions.drv', '/nix/store/s2c5w84jmi0x46lnk5g5qcvqvh3y7qxv-vm-test-run-grub-with-bootspec.drv' failed
```

Same-ish issue when building it locally; I'll just put it x86_64-linux only...

Reply:
I infer this is related to closure size being fixed somewhere in disk images.